draft-ietf-gnap-core-protocol.md

```diff
@@ -2321,18 +2315,15 @@ instance_id (string)
             in the `client` object that the client instance can use in a future request, as
             described in {{request-instance}}.
 
-user_handle (string)
-: A string value used to represent the current
-            user. The client instance can use in a future request, as described in
-            {{request-user-reference}}.
-
+user (string)
+: A string value used to represent the current user, that can be provided to a future request, as described in
+            {{request-user-reference}}. When the end-user is the RO, the handle is equivalent to the opaque subject identifier.   
```

Comment:
I think we should drop this response field entirely and instead only use the `subject` response with an opaque identifier.

Reply:
Thanks Justin. i was hesitant to get that far, but that would be fine for me.